semantic-core/semantic-core.cabal

```diff
@@ -40,53 +40,25 @@ library
   import: common
   hs-source-dirs: src
   exposed-modules:
-    Analysis.Analysis
-    Analysis.Concrete
-    Analysis.File
-    Analysis.FlowInsensitive
-    Analysis.ImportGraph
-    Analysis.ScopeGraph
-    Analysis.Typecheck
-    Control.Carrier.Fail.WithLoc
-    Control.Carrier.Readline.Haskeline
-    Control.Effect.Readline
     Core.Core
-    Core.Core.Parser
-    Core.Core.Pretty
+    Core.Parser
+    Core.Pretty
```

Comment:
I took the opportunity to rename these since they’re unambiguous now by virtue of the module hierarchy change in #327.